plugins/system/debug/src/DataCollector/ProfileCollector.php

```diff
@@ -115,9 +126,9 @@ public function hasStartedMeasure($name): bool
 	 * @param   string  $name    Measurement name.
 	 * @param   array   $params  Parameters
 	 *
-	 * @since  4.0.0
-	 * @throws DebugBarException
 	 * @return void
+	 * @throws DebugBarException
+	 * @since  4.0.0
```

Comment:
https://developer.joomla.org/coding-standards/docblocks.html
correct order and blank lines needed.
```
@return (required, followed by a blank line)
@SInCE (required, followed by a blank line if there are additional tags)
@throws (required if method or function arguments throws a specific type of exception)
```
```suggestion
	 * @return void
	 *
	 * @since  4.0.0
	 *
	 * @throws DebugBarException
```